code/game/objects/auras/regenerating_aura.dm

```diff
@@ -76,7 +76,7 @@
 			var/obj/item/organ/external/E = H.organs_by_name[limb_type]
 			if(E && E.organ_tag != BP_HEAD && !E.vital && !E.is_usable())	//Skips heads and vital bits...
 				if (H.nutrition > grow_threshold)
-					E.removed()			//...because no one wants their head to explode to make way for a new one.
+					H.remove_organ(E) 		//...because no one wants their head to explode to make way for a new one.
```

Comment:
I am opposed to this structural change as discussed on Discord. The organ should handle being removed and then call any mob-specific procs so that the behavior can be easily overridden on organ subtypes.

Reply:
Currently the chain goes like this in order:
/mob/living/carbon/human/remove_organ()
/obj/item/organ/uninstall()
/mob/living/carbon/human/on_lost_organ()
/obj/item/organ/on_removal()

I just don't understand why you want the organs themselves to be the first in the chain? 
It feels like you want organs to just have a wall of "if" checks and type checks to determine what they're connecting to and in what state they are? All that when we got a perfectly good tool to handle avoiding coding case by case in the organ itself?
What does the old structure did that the new one doesn't do better?

Reply:
I linked you my alternative approach on Discord. tldr I don't think there should be a chain like that at all - I think organs, as the atoms that are primarily the point of contact, should handle all that and then pass off to the mob to update as needed. Your code in this PR does stuff like istyping() the organs as they come in as args, which I think is a less than ideal pattern.